examples/02-modal-harmonic/06-cycles_to_failure.py

```diff
@@ -0,0 +1,118 @@
+"""
+DPF-Core Fatigue Engineering Simple Example
+~~~~~~~~~~~~~~~~~~~~
+This example shows how to generate and use a result file to calculate the
+cycles to failure result for a simple model.
+Material data is manually imported, Structural Steel from Ansys Mechanical:
+Youngs Modulus (youngsSteel)
+Poisson's Ratio (prxySteel)
+Cycles to Failure (sn_data)
+The first step is to generate a simple model with high stress and save the
+results .rst locally to myDir (default C:\temp).
+For this we use a short pyMapdl script
+The second step uses DPF-Core to generate the cycles to failure result.
+The locally saved rst is imported and plotted.
+Then the von Mises stress is generated and plotted with DPF operators.
+The python package numpy is then used to interpolate the cycles to failure values.
+The nodal von Mises equivalent stress value is used in the interpolation.
+Note the cycles to failure data needs to be manipulated to use numpy interpolation.
```

Comment:
**Note**: The cycles to failure data must be manipulated to use numpy interpolation.